pkg/controller/scheduledjob/utils.go

```diff
@@ -125,7 +125,11 @@ func getNextStartTimeAfter(schedule string, now time.Time) (time.Time, error) {
 // If there were missed times prior to the last known start time, then those are not returned.
 func getRecentUnmetScheduleTimes(sj batch.ScheduledJob, now time.Time) ([]time.Time, error) {
 	starts := []time.Time{}
-	sched, err := cron.Parse(sj.Spec.Schedule)
+	tmpSched := sj.Spec.Schedule
```

Comment:
Yeah, I'll create one. I've already spoke with cron library author, I'll be addressing those issues in the library, when I'm back from my PTO in Sep.